src/phoenix/server/api/types/Span.py

```diff
@@ -102,6 +102,31 @@ class Span:
     output: Optional[SpanIOValue]
     events: List[SpanEvent]
 
+    @strawberry.field(
+        description="Cumulative token count from self and all "
+        "descendant spans (children, grandchildren, etc.)",
+    )  # type: ignore
+    def cumulative_token_count_total(
```

Comment:
the lazy evaluation here is good but I think we need to sort by it to actually add value. For that don't we need to hoist this computation into the connection resolver? Can be in a follow-up